services/madoc-ts/shim-modules/node_modules/styled-components/index.mjs

```diff
@@ -1,21 +0,0 @@
-import styled from '../../../node_modules/styled-components/dist/styled-components.js';
-
-const defaultStyled = typeof styled === 'function' ? styled : styled.default;
-
-export { defaultStyled as default };
-
-export const ServerStyleSheet = styled.ServerStyleSheet;
-export const StyleSheetConsumer = styled.StyleSheetConsumer;
-export const StyleSheetContext = styled.StyleSheetContext;
-export const StyleSheetManager = styled.StyleSheetManager;
-export const ThemeConsumer = styled.ThemeConsumer;
-export const ThemeContext = styled.ThemeContext;
-export const ThemeProvider = styled.ThemeProvider;
-export const __PRIVATE__ = styled.__PRIVATE__;
-export const createGlobalStyle = styled.createGlobalStyle;
-export const css = styled.css;
-export const isStyledComponent = styled.isStyledComponent;
-export const keyframes = styled.keyframes;
-export const useTheme = styled.useTheme;
-export const version = styled.version;
-export const withTheme = styled.withTheme;
```

Comment:
Why was this deleted?

Reply:
Ah, I had to delete my node modules in rebuild everything after adding tanstack tables, this must of been